templates/paragraphs/paragraph--localgov-link.html.twig

```diff
@@ -45,7 +45,8 @@
     'paragraph',
     'paragraph--type--' ~ paragraph.bundle|clean_class,
     view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
-    not paragraph.isPublished() ? 'paragraph--unpublished'
+    not paragraph.isPublished() ? 'paragraph--unpublished',
+    paragraph.localgov_link_style.value ? 'link--button-style'
```

Comment:
Let's change this, incase we want to add more options, such as 'button-large' or 'button-accent-alt'.

1. Change `paragraph.localgov_link_style.value ? 'link--button-style'` to `paragraph.localgov_link_style.value ? 'link--' ~ paragraph.localgov_link_style.value ~ 'style'
2. Move this up, so that it is directly below the `link` class.